modules/forwarder/lambda.tf

```diff
@@ -11,10 +11,11 @@ resource "aws_lambda_function" "this" {
 
   environment {
     variables = merge({
-      DESTINATION_URI        = "s3://${var.destination.bucket}/${var.destination.prefix}"
-      MAX_FILE_SIZE          = var.max_file_size
-      CONTENT_TYPE_OVERRIDES = join(",", [for o in var.content_type_overrides : "${o["pattern"]}=${o["content_type"]}"])
-      SOURCE_BUCKET_NAMES    = join(",", var.source_bucket_names)
+      DESTINATION_URI             = "s3://${var.destination.bucket}/${var.destination.prefix}"
+      MAX_FILE_SIZE               = var.max_file_size
+      CONTENT_TYPE_OVERRIDES      = join(",", [for o in var.content_type_overrides : "${o["pattern"]}=${o["content_type"]}"])
+      SOURCE_BUCKET_NAMES         = join(",", var.source_bucket_names)
+      OTEL_EXPORTER_OTLP_ENDPOINT = var.debug_endpoint
```

Comment:
I don't see an option to specify any token or credentials for this endpoint. Is this an Observe endpoint or any generic OTLP endpoint?

Reply:
It can be any OTLP compatible endpoint.

Reply:
Fair, but assuming our customers / prospects are primary consumers of this, we will likely get asked "where do I put my datastream token for this endpoint?". Do we have an answer for that?

Reply:
You put it in via basic auth. This can get documented separately.